### PR TITLE
Add status descriptors to CSV

### PR DESCRIPTION
**Pull request by eemcmullan**
`operator-sdk scorecard bundle --selector=test=olm-status-descriptors-test` now passes.

### CODE REVIEW

**Comment by weshayutin** [comment]
NICE!

operator-sdk scorecard bundle --selector=test=olm-status-descriptors-test  --kubeconfig /home/whayutin/.agnosticd/wdh0211pelorusshared/ocp4-cluster_wdh0211pelorusshared_kubeconfig  -n openshift-adp &> /tmp/oadp_head.txt 

change branch...

operator-sdk scorecard bundle --selector=test=olm-status-descriptors-test  --kubeconfig /home/whayutin/.agnosticd/wdh0211pelorusshared/ocp4-cluster_wdh0211pelorusshared_kubeconfig  -n openshift-adp &> /tmp/oadp_580.txt 

[whayutin@thinkdoe oadp-operator]$ git diff /tmp/oadp_head.txt /tmp/oadp_580.txt
diff --git a/tmp/oadp_head.txt b/tmp/oadp_580.txt
index 6fbf2df..c6522f8 100644
--- a/tmp/oadp_head.txt
+++ b/tmp/oadp_580.txt
@@ -6,21 +6,8 @@ Labels:
        "test":"olm-status-descriptors-test"
 Results:
        Name: olm-status-descriptors
-       State: fail
+       State: pass
 
-       Errors:
-               dataprotectionapplications.oadp.openshift.io does not have a status descriptor
-               backups.velero.io does not have a status descriptor
-               backupstoragelocations.velero.io does not have a status descriptor
-               deletebackuprequests.velero.io does not have a status descriptor
-               downloadrequests.velero.io does not have a status descriptor
-               podvolumebackups.velero.io does not have a status descriptor
-               podvolumerestores.velero.io does not have a status descriptor
-               resticrepositories.velero.io does not have a status descriptor
-               restores.velero.io does not have a status descriptor
-               schedules.velero.io does not have a status descriptor
-               serverstatusrequests.velero.io does not have a status descriptor
-               volumesnapshotlocations.velero.io does not have a status descriptor
        Log:
                Loaded ClusterServiceVersion: oadp-operator.v99.0.0
                Loaded 12 Custom Resources from alm-examples

+2000

File: config/manifests/bases/oadp-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -33,71 +33,276 @@ spec:
       displayName: DataProtectionApplication
       kind: DataProtectionApplication
       name: dataprotectionapplications.oadp.openshift.io
+      statusDescriptors:
+      - description: Conditions defines the observed state of Velero
```
Comment (kaovilai) [inline config/manifests/bases/oadp-operator.clusterserviceversion.yaml:37]: If this were generated from comments then might need to edit [oadp types](https://github.com/openshift/oadp-operator/blob/bbf19c5b48da7f326da38c4bb50aca74d49a4f96/api/v1alpha1/oadp_types.go#L181) such that references to velero is replaced.
```suggestion
      - description: Conditions defines the observed state of DataProtectionApplication
```

**Review by kaovilai — CHANGES_REQUESTED** [review]

File: config/manifests/bases/oadp-operator.clusterserviceversion.yaml (hunk 1 above)
Comment (eemcmullan) [reply config/manifests/bases/oadp-operator.clusterserviceversion.yaml]: @kaovilai Fixed

**Comment by kaovilai** [comment]
Please review minor changes suggested in https://github.com/eemcmullan/oadp-operator/pull/1

**Review by kaovilai — APPROVED** [review]

**Review by shubham-pampattiwar — APPROVED** [review]

**Pull request merged**